lab-doug/lib/storage.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const debug = require('debug')('matchscore:storage');
+const AppError = require('./app-error');
+exports.pool = {};
+
+exports.setItem = (function(schema, item) {
```

Comment:
Why do you use exports in some files and module.exports in others? It's not a problem, but it's something worth thinking about. They behave slightly differently, and while module.exports is always safe to use, if you use a module that has been written badly, exports can be unsafe to use.